README.md

```diff
@@ -79,3 +79,19 @@ Curated list of open source projects which are either created by [IBM Lab Servic
 This project is licensed under the terms of the [Creative Commons Zero (CC0 1.0)](LICENSE) license.
 
 [![CC0](https://i.creativecommons.org/p/zero/1.0/88x31.png)](https://creativecommons.org/publicdomain/zero/1.0/)
+
+
+### BaaSapi - GUI for hyperledger fabric network
+* Lab Services contacts:
+  - Liyong Li <liyong.li@ibm.com>
+* Programming language: Golang (API) and VUEjs (frontend)
+* Platform: Linux
+* Links:
+  - [GitHub](https://github.ibm.com/Liyong-Li/baasapi)
+
+- **Description:**
+  BaaSapi is a lightweight management UI which allows you to easily manage your hyperledger fabric environments (kubernetes).
+  BaaSapi allows you to manage your hyperledger fabric resources (network, organization, nodes, CA, channel, chaincodes and more) !
+  
+Introduction
+https://github.ibm.com/Liyong-Li/baasapi/blob/master/Blockchain_Platform_Solution_v1.0.pdf
```

Comment:
Please add a few sentences as plain text. Any links to external documents (e.g. PDF) go under the `* Links:` section above. But please add a few sentences so that people briefly scanning the file understand what this is all about... Alternatively, just delete the "Introduction"-section alltogether.